config/jail.conf

```diff
@@ -908,6 +908,14 @@ bantime      = 1h
 maxretry     = 1
 findtime     = 1
 
+[minecraft]
+# Vanilla server, 1.16.3
+# Change port and logpath as needed
+port = 25565
+findtime = 20
+maxretry = 5
+bantime = 10d
```

Comment:
All this values are rather a matter of user administrating fail2ban, so they are belonging to local settings, e. g. `jail.local`.
Please remove it.

Reply:
Sure thing.